pyk/src/pyk/ktool/kprove.py

```diff
@@ -1,3 +1,4 @@
+import csv
```

Comment:
Replaced `tabulate` with `csv`. The profile csv file can then be easily turned into a table by external tools. (Maybe we could even leave data processing / aggregation to external tools, and just write the raw profile data).